web/src/components/molecules/MyIntegration/WebhookForm/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Checkbox, { CheckboxOptionType } from "@reearth-cms/components/atoms/Checkbox";
+import Col from "@reearth-cms/components/atoms/Col";
+import Divider from "@reearth-cms/components/atoms/Divider";
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import Row from "@reearth-cms/components/atoms/Row";
+import { useT } from "@reearth-cms/i18n";
+
+const WebhookForm: React.FC = () => {
+  const t = useT();
+  const [form] = Form.useForm();
+
+  const options: CheckboxOptionType[] = [
+    { label: t("Create"), value: "Create" },
+    { label: t("Update"), value: "Update" },
+    { label: t("Delete"), value: "Delete" },
+    { label: t("Accessed by API"), value: "Accessed by API" },
```

Comment:
Please add "Delete" and "Accessed by API" to the i18n translation files

Reply:
@nourbalaha , already added, the value does not need a translation.

Reply:
Ok, I will resolve this conversation then.